### PR TITLE
Upgrade PostgreSQL to version 13 in Dockerfile

### PR DESCRIPTION
**Pull request by mayorova**
The upstream container builds started failing in Quay recently, because PostgreSQL 10 was removed from the [RPM](https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm) that is specified in the [Dockerfile](https://github.com/3scale/porta/blob/b767f57ff5fecc6467677daee5b240a5fadc3ea8/openshift/system/Dockerfile#L24).

PostgreSQL 10 is not supported anymore, and the version that is currently supported with 3scale is PostgreSQL 13 (see [Supported Configurations](https://access.redhat.com/articles/2798521)).

### CODE REVIEW

**Review by akostadinov — COMMENTED** [review]
What version do we officially support? Maybe we need to follow officially supported version.

**Comment by mayorova** [comment]
@akostadinov Yes, it's version 13, and I mentioned it in the PR description :wink:

**Review by akostadinov — APPROVED** [review]

**Comment by akostadinov** [comment]
tl;dr; :angel:
